tests/openvino/test_modeling.py

```diff
@@ -1134,6 +1134,10 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
         "decilm",
     )
 
+    SUPPORTED_SSM_ARCHITECTURES = ("mamba", "falcon-mamba")
+    if is_transformers_version(">=", "4.39"):
```

Comment:
```suggestion
    if is_transformers_version(">=", "4.43"):
```
changed sinсу there is an issue with Autotokenizer for mamba models in early version that is reproduced in slow tests.